login/README.md

```diff
@@ -8,6 +8,7 @@
       * [Configuration Parameter Descriptions](#configuration-parameter-descriptions)
       * [Setting up a Development Environment](#setting-up-a-development-environment)
       * [Setting up a Production Environment](#setting-up-a-production-environment)
+      * [Docker](#docker)
```

Comment:
Same comment regarding the `Docker` step. Can this be merged with `Setting up a Production Environment`?

Reply:
Fixed